src/main/java/picard/arrays/GtcToVcf.java

```diff
@@ -383,7 +385,12 @@ private VariantContext makeVariantContext(Build37ExtendedIlluminaManifestRecord
         if (egtFile.totalScore[egtIndex] == 0.0) {
             builder.filter(InfiniumVcfFields.ZEROED_OUT_ASSAY);
             if (genotype.isCalled()) {
-                throw new PicardException("Found a call on a zeroed out SNP!!");
+                if (DO_NOT_ALLOW_CALLS_ON_ZEROED_OUT_ASSAYS) {
+                    throw new PicardException("Found a call on a zeroed out Assay!!");
```

Comment:
Would it be helpful to dump `genotype` here too?

Reply:
sure.